src/components/modal/ModalAddEdit.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <the-modal>
+    <form class="modal__form">
+      <input-group :title="'Title'">
```

Comment:
If you are not binding a dynamic value (i.e. variable), consider just passing a string without `v-bind` (or a `:` shorthand).
```suggestion
      <input-group title="Title">
```